src/utils/message/web2-helper/web2-helper.message.ts

```diff
@@ -1,5 +1,5 @@
 // Define your message types like that:
-// Key: Message name; Value: Feeder functions emit
+// Key: Message name; Parameter: payload data from sender side, Return Value: Feeder functions emit
```

Comment:
Is this an important documentation for end users? If so, I would suggest breaking up the lines for better readability.

Reply:
It is not related to end-users, it only gives info for the devs who contruct these message interfaces.